dissect/target/plugins/os/windows/wer.py

```diff
@@ -11,6 +11,20 @@
 from dissect.target.target import Target
 
 camel_case_patterns = [re.compile(r"(\S)([A-Z][a-z]+)"), re.compile(r"([a-z0-9])([A-Z])"), re.compile(r"(\w)[.\s](\w)")]
+arraylike_pattern = re.compile(r"([_a-zA-Z][a-zA-Z0-9]*)\[(\d+)\]")
+
+
+def _clean_key(key: str) -> str:
+    def _clean(c):
+        return "_" + key[0].encode("unicode_escape").hex()
```

Comment:
field names that start with an underscore are not allowed in `flow.record`. Consider using the prefix `n_` or something else.